crates/sui-config/tests/snapshot_tests.rs

```diff
@@ -57,9 +57,9 @@ fn populated_genesis_snapshot_matches() {
         .generate_accounts(&mut StdRng::from_seed([0; 32]))
         .unwrap();
     let key: AuthorityKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([0; 32])).1;
-    let worker_key: NetworkKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([0; 32])).1;
-    let network_key: NetworkKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([0; 32])).1;
-    let account_key: AccountKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([0; 32])).1;
+    let worker_key: NetworkKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([1; 32])).1;
+    let network_key: NetworkKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([2; 32])).1;
+    let account_key: AccountKeyPair = get_key_pair_from_rng(&mut StdRng::from_seed([3; 32])).1;
```

Comment:
Lets create a single RNG from a seed at the top of this test and pass an &mut reference to all these places that need RNGs.